config/config.ini

```diff
@@ -1063,8 +1055,6 @@ name = Kristján Valur Jónsson
 [http://blog.kritigodey.com/tag/python/rss]
 name = Kriti Godey
 
-[http://krys.ca/category/python/feed/atom/1.0/]
-name = Krys Wilken
```

Comment:
I could get a different feed URL here, but nothing Python-specific.

Reply:
@tseaver can we remove this one?